content/blog/2019-10-24-skimrv2.md

```diff
@@ -0,0 +1,512 @@
+---
+slug: skimrv2
+title: "(Re)introducing `skimr` v2"
+subtitle: "A year in the life of an open source R project"
+package_version: 2.0
+authors:
+  - name: Michael Quinn
+    url: https://github.com/michaelquinn32
```

Comment:
```suggestion
```
new format for authors requires only this:

```
authors:
  - Michael Quinn
```

And your byline will link to your rOpenSci author page
https://ropensci.org/authors/michael-quinn/
https://ropensci.org/authors/elin-waring/

You can edit the links shown on your author page: [Instructions](https://github.com/ropensci/roweb2#create-an-authors-index-file-for-yourself-author-file)